pkg/controller/instance/instance_controller.go

```diff
@@ -79,7 +79,7 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	//
 	// Define a mapping from the object in the event (OperatorVersion) to one or more objects to
 	// reconcile (Instances). Specifically this calls for a reconciliation of any owned objects.
-	ovMapFn := handler.ToRequestsFunc(
+	operatorVersionEventHandler := handler.ToRequestsFunc(
```

Comment:
```suggestion
	ovEventHandler := handler.ToRequestsFunc(
```
we use o and ov all over... it seems like a reasonable short hand